backend/src/services/certificate-est/certificate-est-fns.ts

```diff
@@ -0,0 +1,22 @@
+import { Certificate, ContentInfo, EncapsulatedContentInfo, SignedData } from "pkijs";
+
+export const convertRawCertsToPkcs7 = (rawCertificate: ArrayBuffer[]) => {
```

Comment:
Use methods from `@peculiar/x509` to reduce room for error; see example at the bottom of the repo [here](https://github.com/PeculiarVentures/x509).

Specifically:

const certs = new x509.X509Certificates([
  new x509.X509Certificate("..."),
  new x509.X509Certificate("..."),
]);

certs.export("base64") // <---

Reply:
discussed via DM